src/main/java/com/patsi/service/RecipeService.java

```diff
@@ -38,60 +39,68 @@ public UUID registerRecipe(Recipe recipe) {
     }
 
     //Update Existing Recipe
-    public void updateRecipe (UUID id, Recipe recipe){
+    public void updateRecipe(UUID id, Recipe recipe) {
         recipeRepository.save(recipe);
     }
 
-    public void updateRecipeIcon (UUID id, byte[] recipeIcon) throws IOException {
-        File f = FileHelper.newFile(IMAGE_PATH + id +".jpg");
-            try(FileOutputStream outputStream = FileHelper.newFileOutputStream(f)){
-                outputStream.write(recipeIcon);
-            }
+    public void updateRecipeIcon(UUID id, byte[] recipeIcon) throws IOException {
+        File f = FileHelper.newFile(IMAGE_PATH + id + ".jpg");
```

Comment:
think you can combine the whole writing file logic into one single method in FileHelper, which accept a file path + name and the byte[].

Reply:
fixed